cranelift/codegen/src/isa/x86/abi.rs

```diff
@@ -859,11 +999,57 @@ fn insert_common_epilogue(
     pos: &mut EncCursor,
     reg_type: ir::types::Type,
     csrs: &RegisterSet,
+    fpr_slot: Option<&StackSlot>,
     isa: &dyn TargetIsa,
     is_last: bool,
     mut cfa_state: Option<CFAState>,
 ) {
     let word_size = isa.pointer_bytes() as isize;
+
+    // Even though instructions to restore FPRs are inserted first, we have to append them after
+    // restored GPRs to satisfy parameter order in the return.
+    let mut restored_fpr_values = alloc::vec::Vec::new();
```

Comment:
Uh? Do we prefix `Vec::new()` this way in other places in this file?

Reply:
we in fact do not! The fully-qualified `Vec` reads a bit odd to me too, but I figured we can hoist the `use` if the number of `Vec`-referrers goes up to two.

Reply:
Plenty of codgen modules have `use alloc::vec::Vec`, though, so I wouldn't mind this being lifted to a `use`.